src/redux/missions/missionThunk.js

```diff
@@ -0,0 +1,11 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+const fetchMission = createAsyncThunk('mission/fetchMissions', async () => {
+  const response = await fetch('https://api.spacexdata.com/v3/missions', {
+    method: 'GET',
+  });
+
+  return response.json();
```

Comment:
- [ ] Storing the entire data coming from an API in the redux store may result in unnecessary overhead and slow down the application. Since we just need pieces of information for our app to work, I suggest you filter the data using `map` before sending it to the redux slice. 🧐️

**Necessary data for the missions component**
![Screenshot from 2023-04-13 11-54-18](https://user-images.githubusercontent.com/17243325/231712787-c9530ee4-0676-417d-8c0a-2ef9778508a0.png)